Scheduler: Update SignalR Service description (#633)
DevExpress/devextreme-demos: JSDemos/Demos/Scheduler/SignalRService/description.md

This demo shows how you can use a <a href="https://dotnet.microsoft.com/apps/aspnet/signalr" target="blank">SignalR</a> service to synchronize appointments across different devices. To emulate such a setup, each Scheduler on this page reads data from its own separate data store. Changes made in one control are repeated in the other and persist until the browser session has expired.

Follow the steps below to implement this functionality. Note again that this demo repeats all steps twice for the two Schedulers. Your project will have a single control and a single store.

> 1. Configure a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/). In this demo, we use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method to do it (part of the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data" target="_blank">DevExtreme.AspNet.data</a> extension).

Reviewer: ```suggestion
1. Configure a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/). In this demo, we use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method (part of the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data" target="_blank">DevExtreme.AspNet.data</a> extension).
```

> This demo shows how you can use a <a href="https://dotnet.microsoft.com/apps/aspnet/signalr" target="blank">SignalR</a> service to synchronize appointments across different devices. Each Scheduler is bound to an individual store to emulate the situation when Schedulers are being edited on two different clients. Changes made in one Scheduler are repeated in the other and persist until the browser session has expired.

Reviewer 2: The second sentence has too much passive voice. And there's some repetition of "client", and "scheduler" in this paragraph. 

```suggestion
This demo shows how you can use a <a href="https://dotnet.microsoft.com/apps/aspnet/signalr" target="blank">SignalR</a> service to synchronize appointments across different devices. To emulate such a setup, each Scheduler on this page reads data from its own separate data store. Changes made in one control are repeated in the other and persist until the browser session has expired.
```

> To implement this functionality, follow the steps below:

Reviewer 2: If you want people to implement that functionality in a real application, they will only have a single scheduler and a single store. It's just that those schedulers and stores will run on different devices. That's why I don't think you should use plural in the instructions below. Our customers don't need to configure multiple data stores. They don't need to call "each" store's push() method. 

So if you want to make this clear, maybe rephrase it like this:

 ```suggestion
Follow the steps below to implement this functionality. Note again that this demo repeats all steps twice for the two Schedulers. Your project will have a single control and a single store.
```

> 1. Configure CustomStores. In this demo, we use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method (part of the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data" target="_blank">DevExtreme.AspNet.data</a> extension).

Reviewer 2: What's a "CustomStore"? I don't see any references on the page (in code).

> 1. Create store instances&mdash;one per Scheduler.

Reviewer 2: In the last step, when I used the `createStore` method, didn't I already create a store instance? Judging by code, I guess what you mean here is "store a reference to the store in a variable". Some implementations differ depending on technology... But in any event it seems that these two steps are saying the same thing.
